colabs/pyg/Graph_Classification_with_PyG_and_W&B.ipynb

```diff
@@ -1,12 +1,13 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    wandb.log(data_details) #🪄🐝
</pre><p>should be optional</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/366/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>